src/libraries/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -539,6 +538,31 @@ public static void CopyConstructorTest_NullInput()
             Assert.Contains("options", ex.ToString());
         }
 
+        [Fact]
+        public static void DefaultSerializerOptions_General()
+        {
+            var options = new JsonSerializerOptions();
+            var newOptions = new JsonSerializerOptions(JsonSerializerDefaults.General);
+            VerifyOptionsEqual(options, newOptions);
+        }
+
+        [Fact]
+        public static void PredefinedSerializerOptions_Web()
+        {
+            var options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
+            Assert.True(options.PropertyNameCaseInsensitive);
+            JsonNamingPolicy policy = options.PropertyNamingPolicy;
+            Assert.NotNull(policy);
+            Assert.Equal(policy, JsonNamingPolicy.CamelCase);
```

Comment:
```suggestion
            Assert.Same(JsonNamingPolicy.CamelCase, options.PropertyNamingPolicy);
```

- `Assert.NotNull` isn't needed here
- The expected value should be the first arg
- `Assert.Same` is used when we expect the same object reference